spec/models/message_spec.rb

```diff
@@ -113,5 +113,17 @@
         end
       end
     end
+
+    describe 'incoming message' do
+      let(:message) { create(:message, sender: create(:contributor), recipient: nil) }
+
+      it 'does not send outbound messages to any channels' do
+        [PostmarkAdapter::Outbound, SignalAdapter::Outbound, TelegramAdapter::Outbound, ThreemaAdapter::Outbound].each do |adapter|
+          expect(adapter).not_to receive(:send!)
```

Comment:
You could also test the happy path here, so we can see that `recipient: nil` makes the difference.